app/models/page.rb

```diff
@@ -15,15 +16,15 @@ class Page
   field :role  # only relevant for virtual pages
   field :active, type: Boolean, default: false
   field :layout, type: String, default: 'default'
-  field :meta_description, localize: Slices::Config.i18n?
-  field :title, localize: Slices::Config.i18n?
+  field :meta_description, localize: Slices::Config.i18n?, default: ''
+  field :title, localize: Slices::Config.i18n?, default: ''
```

Comment:
I've added a link to the bug report in the commit and updated the guides to say a default should be set.